examples/ASMapNode/Sample/MapHandlerNode.m

```diff
@@ -98,13 +98,10 @@ - (void)didLoad
   self.mapNode.imageForStaticMapAnnotationBlock = ^UIImage *(id<MKAnnotation> annotation, CGPoint *centerOffset){
     MapHandlerNode *grabbedSelf = weakSelf;
     if (grabbedSelf) {
-      MKAnnotationView *av = [grabbedSelf annotationViewForAnnotation:annotation];
-      *centerOffset = av.centerOffset;
-      UIGraphicsBeginImageContextWithOptions(av.bounds.size, av.opaque, 0.0);
-      [av.layer renderInContext:UIGraphicsGetCurrentContext()];
-      UIImage *pinImage = UIGraphicsGetImageFromCurrentImageContext();
-      UIGraphicsEndImageContext();
-      return pinImage;
+      if ([annotation isKindOfClass:[CustomMapAnnotation class]]) {
+        MKAnnotationView *av = [grabbedSelf annotationViewForAnnotation:annotation];
```

Comment:
Hmm unfortunately we just cannot allocate views in here, since we're off the main thread. Since CustomMapAnnotation has an image property, let's use that. Also let's make sure to set `centerOffset` to 21,21